openrtb_ext/bidders.go

```diff
@@ -568,10 +565,42 @@ type BidderParamValidator interface {
 	Schema(name BidderName) string
 }
 
+type paramsValidatorHelper interface {
+	readDir(name string) ([]os.DirEntry, error)
+	readFile(name string) ([]byte, error)
+	newReferenceLoader(source string) gojsonschema.JSONLoader
+	newSchema(l gojsonschema.JSONLoader) (*gojsonschema.Schema, error)
+	abs(path string) (string, error)
+}
+
+type paramsHelper struct{}
```

Comment:
In other parts of PBS, if we have an interface `foo` the main implementation is `standardFoo`.

Reply:
https://github.com/prebid/prebid-server/pull/3048/commits/8e4cd4c9252d8b836b76b0833cb201c24d58aa3a